osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/resources/db/migration/V20200127155743442__Initial_version_PostgreSQL.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE mqtt_device (
+    id bigint NOT NULL,
+    creation_time timestamp without time zone NOT NULL,
+    modification_time timestamp without time zone NOT NULL,
+    device_identification character varying(40) NOT NULL,
+    host character varying(255),
+    port integer,
+    topics character varying(255),
+    qos character varying(255),
+    version bigint
```

Comment:
Make sure this is the 4th colum, just like all other entities.

Reply:
Done